exonum/src/runtime/rust/stubs.rs

```diff
@@ -108,3 +127,110 @@ impl GenericCall<InstanceId> for (PublicKey, SecretKey) {
         Verified::from_value(tx, self.0, &self.1)
     }
 }
+
+#[cfg(test)]
+mod explanation {
```

Comment:
Love this explanation. Now it's much easier to "unravel the magic" :)

Reply:
What do you think to make this module more accessible, for example, after I put the Rust runtime into a separate crate&

Reply:
I think we'll decide on that later. Maybe we'll add a whole brand new article about this approach to the docs at `exonum.com`. But for now that's more than enough I believe :)